package.json

```diff
@@ -28,12 +28,19 @@
     "shutdown",
     "sigterm",
     "sigint",
+    "sighup",
     "terminate",
     "kill",
     "stop",
     "event"
   ],
   "devDependencies": {
-    "ava": "0.0.4"
+    "ava": "0.13.0"
+  },
+  "ava": {
+    "files": [
+      "test/*.js",
+      "!tests/cases/*"
```

Comment:
Just `cases` => `fixtures` and you won't need to define any files here, as AVA ignores `fixtures` folders.